x/ibc/20-transfer/handler.go

```diff
@@ -0,0 +1,25 @@
+package transfer
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/types"
+)
+
+// HandleMsgTransfer defines the sdk.Handler for MsgTransfer
+func HandleMsgTransfer(ctx sdk.Context, k Keeper, msg MsgTransfer) (res sdk.Result) {
+	err := k.SendTransfer(ctx, msg.SrcPort, msg.SrcChannel, msg.Denomination, msg.Amount, msg.Sender, msg.Receiver, msg.Source)
+	if err != nil {
+		return err.Result()
```

Comment:
I think there's a `sdkerrors.ResultFromError(err)` function that we can use instead